troposphere/static/js/components/admin/AtmosphereUser.jsx

```diff
@@ -60,7 +61,7 @@ export default React.createClass({
             <td style={{ border: "none" }} 
                 className="user-name"
             >
-                {user.get("username")}
+                {user.get("username")} <Emulate username={user.get("username")} />
```

Comment:
I know this is for the admin panel, so not user facing. You can take it or leave, but...

A good amount of whitespace between the user name and icon would help polish this up. Around 20px of margin would make the difference.

Not sure how wide these user cells get but if the gap isn't too wide, if you aligned the icons to the right side of the cell they would all line up rather than jagged as they are now.
<img width="419" alt="screen shot 2017-06-19 at 6 40 47 pm" src="https://user-images.githubusercontent.com/7366338/27312703-e1d1f81a-551e-11e7-839e-b94d51b53bb2.png">